README.md

```diff
@@ -2,12 +2,41 @@
 
 [![build status](https://img.shields.io/travis/enkidevs/approve-ci/master.svg?style=flat-square)](https://travis-ci.org/enkidevs/approve-ci)
 
-https://developer.github.com/guides/building-a-ci-server/
+Approve-ci monitors pull requests and checks for approval. Once the number of approvals exceeds the number of disapprovals by a set amount the pull request is marked as 'approved' and is ready to merge.
 
-https://developer.github.com/v3/repos/hooks/
+![approve-ci in use](http://i.imgur.com/2aMhuzk.png)
 
-https://developer.github.com/v3/activity/events/types/#pullrequestreviewcommentevent
+## Deployment
 
-https://github.com/mikedeboer/node-github
+To run an approve-ci bot clone this repository, modify `.approve-ci` and deploy your application. You must set a GitHub organisation, repository and an API [token](https://github.com/settings/tokens) with `repo` rights as environment variables, along with the URL of your deployment.
 
-https://github.com/facebook/mention-bot
+```
+  export GITHUB_TOKEN='f7c41472410cacded24090d24f70e98995d8dc55'
+  export GITHUB_REPO='approve-ci'
+  export GITHUB_ORG='enkidevs'
+  export URL='http://approve-ci.herokuapp.com/'
+```
+
+Alternatively you can deploy the bot using Heroku by pressing the button below:
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
+GitHub allows you [protect branches](https://github.com/enkidevs/approve-ci/settings/branches) and to require specific tests to pass before pull requests can be merged. By setting this up and specifying the approve-ci bot (the name is defined in the configuration file, see the next section), approval is needed before a request can be merged.
```

Comment:
people don't have access to https://github.com/enkidevs/approve-ci/settings/branches. would be better to show the link https://github.com/USERNAME/REPO/settings/branches